autocoder/ai.py

```diff
@@ -1,3 +1,5 @@
+from typing import Iterator
+from langchain.schema import BaseMessageChunk
```

Comment:
@devbuddytest I think the correct import is `from langchain.schema.messages import BaseMessageChunk`